src/Symfony/Component/Security/Http/Firewall/RememberMeListener.php

```diff
@@ -34,18 +35,20 @@ class RememberMeListener implements ListenerInterface
     private $logger;
     private $dispatcher;
     private $catchExceptions = true;
+    private $sessionStrategy;
 
     /**
      * Constructor.
      *
-     * @param TokenStorageInterface          $tokenStorage
-     * @param RememberMeServicesInterface    $rememberMeServices
-     * @param AuthenticationManagerInterface $authenticationManager
-     * @param LoggerInterface                $logger
-     * @param EventDispatcherInterface       $dispatcher
-     * @param bool                           $catchExceptions
+     * @param TokenStorageInterface                  $tokenStorage
+     * @param RememberMeServicesInterface            $rememberMeServices
+     * @param AuthenticationManagerInterface         $authenticationManager
+     * @param LoggerInterface                        $logger
+     * @param EventDispatcherInterface               $dispatcher
+     * @param bool                                   $catchExceptions
+     * @param SessionAuthenticationStrategyInterface $sessionStrategy
      */
-    public function __construct(TokenStorageInterface $tokenStorage, RememberMeServicesInterface $rememberMeServices, AuthenticationManagerInterface $authenticationManager, LoggerInterface $logger = null, EventDispatcherInterface $dispatcher = null, $catchExceptions = true)
+    public function __construct(TokenStorageInterface $tokenStorage, RememberMeServicesInterface $rememberMeServices, AuthenticationManagerInterface $authenticationManager, LoggerInterface $logger = null, EventDispatcherInterface $dispatcher = null, $catchExceptions = true, SessionAuthenticationStrategyInterface $sessionStrategy = null)
```

Comment:
The `$catchExceptions` argument is not in 2.3, so I'm going to merge this one on 2.8 instead.